subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AbstractSmokeTest.groovy

```diff
@@ -28,6 +28,7 @@ import static org.gradle.api.internal.artifacts.BaseRepositoryFactory.*
 import static org.gradle.integtests.fixtures.RepoScriptBlockUtil.*
 
 abstract class AbstractSmokeTest extends Specification {
+    private static final String MIRROR_INIT_SCRIPT_LOCATION = "org.gradle.smoketests.mirror.init.script"
```

Comment:
I am wondering if we want to tie that property to being something related to our mirrors. Maybe we should just call this thing `org.gradle.smoketests.init.script`, so we can later on pass a possibly different init script with the same property. WDYT?

Reply:
Good idea.